hadoop-ozone/ozone-recon/src/main/java/org/apache/hadoop/ozone/recon/spi/ContainerDBServiceProvider.java

```diff
@@ -85,7 +86,8 @@ Integer getCountForForContainerKeyPrefix(
    * @return Map of containerID -> containerMetadata.
    * @throws IOException
    */
-  Map<Long, ContainerMetadata> getContainers(int limit) throws IOException;
+  Map<Long, ContainerMetadata> getContainers(int limit, long start)
```

Comment:
According to the implementation, the start key will be skipped if present in the seek. This is to support a pagination kind of API. Maybe, we can change the param name to reflect this. Something like previous instead of start?